package.json

```diff
@@ -77,6 +74,7 @@
 		"postcss": "^8.4.38",
 		"postcss-loader": "^8.1.1",
 		"postcss-scss": "^4.0.9",
+		"prettier": "^3.3.2",
```

Comment:
Remove all Prettier-related things, it wouldn't be too wise to add them to the existing setup (there would probably be a loooot of re-formatting 😄)